source/code/source/playfab/QoS/PlayFabQoSApi.cpp

```diff
@@ -247,7 +247,7 @@ namespace PlayFab
             // Setup sockets based on the number of threads
             for (unsigned int i = 0; i < numThreads; ++i)
             {
-                shared_ptr<QoSSocket> socket = shared_ptr<QoSSocket>(new QoSSocket());
+                shared_ptr<QoSSocket> socket = make_shared<QoSSocket>();
```

Comment:
>make_shared<QoSSocket>(); [](start = 47, length = 25)

Change if you want, but i think we were supporting Std-11 until now in Xplat SDK

Reply:
This isn't the first make_shared in the published/central part of the SDK.
We've not been testing C++11, so many make_shared's have already found their way into the SDK.
We could probably restore C++11 if customers declare the need, but for now, the ship has already sailed.

---
In reply to: [355050637](https://github.com/PlayFab/XPlatCoreTemplate/pull/60#discussion_r355050637) [](ancestors = 355050637)